wordssite/templates/dictionary/word_detail.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+<div class="container mt-5" style="width: 70%;">
+    <div class="col">
+        <h1 class="text-center">"{{word}}" translations:</h1>
+        <div class="d-flex justify-content-center">
+            <table class="table table-striped mt-5" style="width: 50%">
+                <tbody>
+                    {% for translation in word.translation_set.all %}                    <tr>
+                        {% comment %} <th scope="row">{{forloop.counter}}</th> {% endcomment %}
```

Comment:
what does this line do?

Reply:
for loop that iterates over the list of all the translations that are available for a word.
` {% comment %} <th scope="row">{{forloop.counter}}</th> {% endcomment %}` was copied from the original tutorial  -deleted we are not using it